app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file, new_file = command.split("")
```

Comment:
As Yurii said, "no need to use space as arg in split method, it uses space by default"